cmd/gateway-azure.go

```diff
@@ -125,14 +126,19 @@ func azureToObjectError(err error, params ...string) error {
 }
 
 // Inits azure blob storage client and returns AzureObjects.
-func newAzureLayer(args []string) (GatewayLayer, error) {
-	endPoint, secure, err := parseGatewayEndpoint(args[0])
-	if err != nil {
-		return nil, err
-	}
+func newAzureLayer(args cli.Args) (GatewayLayer, error) {
+
+	var err error
 
-	if endPoint == "" {
-		endPoint = storage.DefaultBaseURL
+	// Default endpoint parameters
+	endPoint := storage.DefaultBaseURL
+	secure := true
+
+	if len(args) > 0 {
```

Comment:
Updated with args.First(). Let's see as discussed if we need to check if there is another way to provide default parameters.